gdal/src/main/scala/geotrellis/raster/gdal/GDALRasterSource.scala

```diff
@@ -110,7 +110,7 @@ class GDALRasterSource(
       }
   }
 
-  def reprojection(targetCRS: CRS, resampleTarget: ResampleTarget = DefaultTarget, method: ResampleMethod = NearestNeighbor, strategy: OverviewStrategy = OverviewStrategy.DEFAULT): RasterSource =
+  def reprojection(targetCRS: CRS, resampleTarget: ResampleTarget = DefaultTarget, method: ResampleMethod = ResampleMethod.DEFAULT, strategy: OverviewStrategy = OverviewStrategy.DEFAULT): RasterSource =
```

Comment:
Decided to replace all `NearestNeighbor` calls with `ResampleMethod.DEFAULT` in all RasterSources.
`ResampleMethod.DEFAULT == NearestNeighbor`